Sources/CoreMetrics/Metrics.swift

```diff
@@ -23,8 +23,10 @@
 ///
 /// Its behavior depends on the `CounterHandler` implementation.
 public final class Counter {
-    /// ``_handler`` is only public to allow access from `MetricsTestKit`. Do not consider it part of the public API.
+    /// `_handler` and `_factory` are only public to allow access from `MetricsTestKit`.
+    /// Do not consider them part of the public API.
     public let _handler: CounterHandler
+    public let _factory: MetricsFactory
```

Comment:
Can't this be `package` instead then?

Reply:
Yup, I mirrored the `_handler` here, filed an issue to fix it up across the board separately: https://github.com/apple/swift-metrics/issues/173